contracts/monitoring/src/contract.rs

```diff
@@ -73,14 +69,78 @@ mod tests {
         let info = mock_info("instantiator", &[]);
         let env = mock_env();
 
-        let msg = InstantiateMsg {
-            governance_address: governance.to_string(),
-        };
-
-        let res = instantiate(deps.as_mut(), env, info, msg);
+        let res = instantiate(
+            deps.as_mut(),
+            env,
+            info,
+            InstantiateMsg {
+                governance_address: governance.to_string(),
+            },
+        );
         assert!(res.is_ok());
 
         let config = CONFIG.load(deps.as_ref().storage).unwrap();
         assert_eq!(config.governance, governance);
     }
+
+    #[test]
+    fn test_add_prover_from_goverance() {
```

Comment:
don't start with `test_`. The functions are already clearly labeled with the test attribute. Instead add the expected outcome